packages/core/deprecations/core-deprecations-server-internal/README.md

```diff
@@ -1,3 +1,90 @@
 # @kbn/core-deprecations-server-internal
 
 This package contains the internal types and implementation of Core's server-side `deprecations` service.
+
+
+/** Router:
```

Comment:
We should move the guide to the public types package. Internal core packages shouldn't be consumed outside of core.